projects/1.16/assets/growthcraft-community-edition/growthcraft_milk/lang/zh_cn.json

```
@@ -0,0 +1,100 @@
+   "item.growthcraft_milk.butter_milk_bucket": "酪乳桶",
+   "item.growthcraft_milk.butter_salted": "咸黄油",
+   "item.growthcraft_milk.casu_marzu_cheese_curds_drained": "卡苏马苏活蛆奶酪凝乳（挤干）",
+   "item.growthcraft_milk.casu_marzu_cheese_slice": "卡苏马苏活蛆奶酪切块",
+   "item.growthcraft_milk.cheddar_cheese_curds_drained": "切达奶酪凝乳（挤干）",
+   "item.growthcraft_milk.cheddar_cheese_slice": "切达奶酪切块",
+   "item.growthcraft_milk.cheese_base_bucket": "奶酪基料桶",
+   "item.growthcraft_milk.cheese_cloth": "奶酪纱布",
+   "item.growthcraft_milk.condensed_milk_bucket": "炼乳桶",
+   "item.growthcraft_milk.cream_bucket": "奶油桶",
+   "item.growthcraft_milk.cultured_milk_bucket": "酸奶桶",
+   "item.growthcraft_milk.emmentaler_cheese_curds_drained": "门塔尔奶酪凝乳（挤干）",
+   "item.growthcraft_milk.emmentaler_cheese_slice": "门塔尔奶酪切块",
+   "item.growthcraft_milk.gorgonzola_cheese_curds_drained": "戈尔贡佐拉奶酪凝乳（挤干）",
+   "item.growthcraft_milk.gorgonzola_cheese_slice": "戈尔贡佐拉奶酪切块",
+   "item.growthcraft_milk.gouda_cheese_curds_drained": "豪达奶酪凝乳（挤干）",
+   "item.growthcraft_milk.gouda_cheese_slice": "豪达奶酪切块",
+   "item.growthcraft_milk.ice_cream_apple": "苹果冰激凌",
+   "item.growthcraft_milk.ice_cream_chocolate": "巧克力冰激凌",
+   "item.growthcraft_milk.ice_cream_grape_purple": "葡萄冰激凌",
+   "item.growthcraft_milk.ice_cream_grape_red": "葡萄冰激凌",
+   "item.growthcraft_milk.ice_cream_grape_white": "葡萄冰激凌",
+   "item.growthcraft_milk.ice_cream_honey": "蜂蜜冰激凌",
+   "item.growthcraft_milk.ice_cream_pumpkin": "南瓜香料冰激凌",
+   "item.growthcraft_milk.ice_cream_watermelon": "西瓜冰激凌",
+   "item.growthcraft_milk.kumis_bucket": "马奶酒桶",
+   "item.growthcraft_milk.milk_bucket": "牛奶桶",
+   "item.growthcraft_milk.milking_bucket_iron": "铁奶桶",
+   "item.growthcraft_milk.monterey_cheese_curds_drained": "蒙特瑞奶酪凝乳（挤干）",
+   "item.growthcraft_milk.monterey_cheese_slice": "蒙特瑞奶酪切块",
+   "item.growthcraft_milk.parmesan_cheese_curds_drained": "帕马森奶酪凝乳（挤干）",
+   "item.growthcraft_milk.parmesan_cheese_slice": "帕马森奶酪切块",
+   "item.growthcraft_milk.provolone_cheese_curds_drained": "波萝伏洛奶酪凝乳（挤干）",
+   "item.growthcraft_milk.provolone_cheese_slice": "波萝伏洛奶酪切块",
+   "item.growthcraft_milk.rennet_bucket": "凝乳酶桶",
+   "item.growthcraft_milk.ricotta_cheese_curds_drained": "里科塔奶酪凝乳（挤干）",
+   "item.growthcraft_milk.ricotta_cheese_slice": "里科塔奶酪碗",
+   "item.growthcraft_milk.skim_milk_bucket": "脱脂牛奶桶",
+   "item.growthcraft_milk.starter_culture": "发酵剂培养物",
+   "item.growthcraft_milk.stomach": "胃",
+   "item.growthcraft_milk.thistle": "蓟",
+   "item.growthcraft_milk.thistle_seed": "蓟种子",
+   "item.growthcraft_milk.whey_bucket": "乳清桶",
+   "item.growthcraft_milk.yogurt_apple": "苹果酸奶",
+   "item.growthcraft_milk.yogurt_chocolate": "巧克力酸奶",
+   "item.growthcraft_milk.yogurt_grape_purple": "紫葡萄酸奶",
+   "item.growthcraft_milk.yogurt_grape_red": "红葡萄酸奶",
+   "item.growthcraft_milk.yogurt_grape_white": "白葡萄酸奶",
+   "item.growthcraft_milk.yogurt_honey": "蜂蜜酸奶",
+   "item.growthcraft_milk.yogurt_plain": "原味酸奶",
+   "item.growthcraft_milk.yogurt_pumpkin": "南瓜香料酸奶",

```

```suggestion
   "item.growthcraft_milk.yogurt_pumpkin": "南瓜酸奶",
```
`南瓜拿铁`/`Pumpkin Spice Latte`